business/data-confidentiality.md

```diff
@@ -4,12 +4,35 @@ description: Data confidentiality in machine translation
 
 # Data confidentiality
 
-**Data confidentiality** refers to how the machine translation providers use the specific data sent for translation. Providers can generate translations with or without storing the input. Machine translation providers need data to train and improve their translation service.
+**Data confidentiality** terms or policies govern how machine translation providers use client data.
 
-In systems with no confidentiality settings, input data can reappear in similar machine translation calls. Providers that offer no-trace never store data.
+Machine translation providers use client data to train and improve their translation systems. Machine translation clients require data confidentiality for:
 
-Free machine translation services have lower confidentiality levels than private machine translation services.
+- Compliance with regulations and standards like HIPAA and GDPR
+- Compliance with security reviews
+- Competitiveness
```

Comment:
I guess "Competitiveness" can be removed, because that's why security guidelines exist.

But I meant, there are 3 distinct issues:
- consumer user data
- their B2B client data
- their own business data (e.g. new product launches)